compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala

```diff
@@ -141,6 +143,8 @@ class PlainPrinter(_ctx: Context) extends Printer {
         ParamRefNameString(tp) ~ ".type"
       case tp: TypeParamRef =>
         ParamRefNameString(tp) ~ lambdaHash(tp.binder)
+      case tp: AppliedTermRef =>
+        toTextRef(tp) ~ ".type"
```

Comment:
shouldn't you wrap in `{` and `}` to reflect the syntax instead?

Reply:
Indeed, though it does require some state to track when we hit the first `AppliedTermRef`. Is it okay to keep this kind of flag in `PlainPrinter` itself?

Reply:
I guess so, we already have state for rectypes.